### ISSUE TITLE
Multiline input NVDA screen reader cannot read out clipped text

### ISSUE THREAD

**Issue by kingszi** [jpmorganchase/salt-ds#4824]
### Latest version

- [x] I have tested the latest version

### Description

When tabbing onto a MultilineInput within a FormField using a keyboard with a screen reader, the screen reader is not able to read out the clipped text even if using the up and down arrow keys to navigate over to the clipped text. 
Unsure if this is a bug, I might be missing something but was unable to see anything on this in the documentation.

### Steps to reproduce

1. Turn on NVDA Screen Reader
2. MultilineInput within a FormField with clipped text -> tab onto it
3. press Down arrow key to navigate over to clipped text
4. Screen reader still reads out first visible line

### Package name(s)

Core (@salt-ds/core)

### Package version(s)

1.39.0, 1.42.0

### Browser

- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Microsoft Edge

### Operating system

- [ ] macOS
- [x] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.

**Comment by jake-costa** [comment]
@kingszi Thanks for reaching out. I am unable to reproduce this issue you are encountering. 

I have a few questions:
1. You mention "press Down arrow key to navigate over to clipped text, have you also tried NVDA Key + arrow key?
2. Can you reproduce it in [Salt storybook](https://storybook.saltdesignsystem.com/?path=/story/core-multiline-input--default)?
3. Are you using a Laptop or desktop?
4. While it should switch automatically, have you tried to manually switch modes (Browse/Focus) NVDA Key + Space


Note: NVDA Key can be assigned in the settings but is usually either insert or caps lock.

**Comment by kingszi** [comment]
Hi @jake-costa  thank you so much for the detailed response.
I was unable to reproduce this in the Salt storybook, so went through our code and I think the issue was causes by the 'rows' prop being passed in incorrectly.
In our app I am still unable to just use the Arrow keys but it seems to work well with using the NVDA key now which wasn't the case before.
Thank you again for your help, I will close this issue.

**Comment by kingszi** [comment]
Hi @jake-costa , I have actually been able to reproduce this in the storybook as well - it seems to happen when the multiline input field is read-only, and the text overflows the allowed rows (e.g rows is set to 2 and input text is 4 lines), in this case the screen reader will only read out the first visible line. I have tried with the NVDA key + arrow key and NVDA key + Space also and it doesn't seem to work very smoothly. Would you be able to advise if you can see the same?

**Comment by jake-costa** [comment]
Hi @kingszi, 
Thanks for reaching out. I am unable to reproduce this in storybook or my local dev environment.

You mention that NVDA key + Space doesn't work but just to confirm, by doing the following you are not able to get NVDA to read content in the read-only textarea:
1. Tab to the textarea (NVDA should announce you are on a read-only multiline input and announce the visible text)
2. Change modes (Browse mode to Focus mode) using NVDA key+ Space. You will hear a sound to confirm you have switched modes successfully).
3. Then use arrow keys to navigate the additional content in the textarea.

For context:
NVDA reads only the first visible line of multi-line read-only inputs (like <textarea>) in browse mode because it treats these fields as interactive elements, even if they’re not editable. Browse mode essentially lets users skim pages and gives the user a quick preview of the element (first visible line) to avoid overwhelming users with long text blocks. To hear the full content, you need to switch to focus mode, which allows arrow-key navigation through all lines. NVDA does this intentionally as it prioritizes skimming in browse mode and reserves detailed exploration for focus mode. 

If you are still running into issues please provide the detailed steps to reproduce this issue in storybook and I will look into it.

Resources:
[NVDA User Guide Browse Mode](https://download.nvaccess.org/documentation/userGuide.html#BrowseMode)

**Comment by kingszi** [comment]
Hi @jake-costa , thank you again for taking the time to explain this. 
Following your steps above, when I get to step 3, the arrow keys seem to navigate between the fields on our form rather than within the text area, but once I navigate away and then back with the arrow keys I can hear the whole text being read out. 
I think this is probably just me not being familiar with the usage of NVDA keys rather than a bug. Thanks again!
